📄 readme edits (#384)
mondoohq/cnspec: README.md

**Open source, cloud-native security and policy project**

> cnspec assesses your entire infrastructure's security and compliance. It finds vulnerabilities and misconfigurations on all your systems, including: public and private cloud environments, Kubernetes clusters, containers, container registries, servers, and endpoints, SaaS products, infrastructure as code, APIs, and more.

Reviewer: systems isn't the right word since we're covering SaaS as well

Author: ```suggestion
cnspec assesses your entire infrastructure's security and compliance. It finds vulnerabilities and misconfigurations throughout public and private cloud environments, Kubernetes clusters, containers, container registries, servers, and endpoints, SaaS products, infrastructure as code, APIs, and more.
```
